Makefile.am

```diff
@@ -278,22 +285,15 @@ release:
 	@git tag -a -m "$(PACKAGE_NAME) release $(PACKAGE_VERSION)" v$(PACKAGE_VERSION)
 	@printf "\nNew release $(PACKAGE_VERSION) tagged!\n\n"
 
-
-MANPAGES = \
-	docs/swupd.1 \
-	docs/check-update.service.4 \
-	docs/check-update.timer.4 \
-	docs/swupd-update.service.4 \
-	docs/swupd-update.timer.4 \
-	docs/update-triggers.target.4
-
-dist_man_MANS = \
-	$(MANPAGES)
-
 man: $(dist_man_MANS)
 
 .rst:
 	mkdir -p "$$(dirname $@)"
 	rst2man.py "$<" > "$@.tmp" && mv -f "$@.tmp" "$@"
 
-test/functional/completion/completion-basic.bats: swupd.bash
```

Comment:
Did you intend to remove this line?

Reply:
yes. I should have added a comment about that in the commit message.
swupd.bash was autogenerated a long time ago, but it isn't anymore, so we don't need this line

Reply:
Okay, thanks.